pyxdameraulevenshtein/pyxdameraulevenshtein.pyx

```diff
@@ -23,7 +23,7 @@
 
 from cpython.version cimport PY_MAJOR_VERSION
 from libc.stdlib cimport calloc, free
-import numpy as np
```

Comment:
@gfairchild The `cimport` should have taken care of most of the `numpy` interfaces as exposed [here](https://github.com/cython/cython/blob/master/Cython/Includes/numpy/__init__.pxd). However, the `vectorize` function doesn't appear to be available, hence we'll need the `import` for that. I forgot to re-cythonize, hence missed this. I'll revert this hunk.